packit_service/worker/parser.py

```diff
@@ -1544,3 +1544,15 @@ def parse_vm_image_build_result_event(event) -> Optional[VMImageBuildResultEvent
             message,
             created_at,
         )
+
+    MAPPING = {
+        "github": {
+            "check_run": parse_check_rerun_event,
+            "pull_request": parse_pr_event,
+            "pull_request_review_comment": parse_pull_request_comment_event,
+            "issue_comment": parse_issue_comment_event,
```

Comment:
Good catch! Yes, it can, I completely missed it in the [documentation](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#issue_comment).

The issue in the code above is that I mistakenly assumed, that we are listening to `pull_request_review_comment` events, but we are not! We are not even subscribed to those! :smile:

Instead, `issue_comment` events are parsed by `parse_issue_comment_event()` if they don't have a `issue.pull_request` field, and by `parse_pull_request_comment_event()` if they have.

Reply:
Fixed this by introducing `parse_github_comment_event()`. PTAL.